### PR TITLE
Remove MinatarEnv and use native Minatar gym wrapper

### PR DESCRIPTION
**Pull request by dapatil211**

### CODE REVIEW

**Comment by sriyash421** [comment]
Merge conflict here. @dapatil211 miniatar.py is deleted in this pr and dev has some updates to it.

File: hive/configs/gym/random.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# General training loop config
+run_name: &run_name 'gym-random'
```
Comment (alirahkay) [inline hive/configs/gym/random.yml]: Please only put the PR-related changes into your PR. :)

File: hive/configs/minatar/rainbow_circular_buffer.yml [hunk 2]
```diff
@@ -15,9 +15,9 @@ save_dir: 'experiment'
 
 # Environment config
 environment:
-  name: 'MinAtarEnv'
+  name: 'GymEnv'
   kwargs:
-    env_name: 'breakout'
+    env_name: 'Breakout-v0'
```
Comment (alirahkay) [inline hive/configs/minatar/rainbow_circular_buffer.yml]: I think this should be `MinAtar/Breakout-v0`. Am I right?

Based on the instructions given [here](https://github.com/kenjyoung/MinAtar#openai-gym-wrapper).

**Review by alirahkay — CHANGES_REQUESTED** [review]

File: hive/configs/minatar/dqn2.yml [hunk 3]
```diff
@@ -0,0 +1,92 @@
+# General training loop config
```
Comment (alirahkay) [inline hive/configs/minatar/dqn2.yml]: Also, what is the purpose of this file?

**Comment by dapatil211** [comment]
@alirahkay @sriyash421 , Fixed your concerns. Now this PR only deals with Minatar.

**Review by alirahkay — APPROVED** [review]
LGTM!

**Review by sriyash421 — APPROVED** [review]

**Pull request merged**